libmamba/src/core/prefix_data.cpp

```diff
@@ -261,11 +267,11 @@ namespace mamba
         {
             j = nlohmann::json::parse(out);
```

Comment:
I think this may be useful as well:
```suggestion
            j = nlohmann::json::parse(util::strip(out));
```